src/signature.c

```diff
@@ -1212,10 +1212,9 @@ static int p11prov_rsasig_digest_sign_final(void *ctx, unsigned char *sig,
 {
     P11PROV_SIG_CTX *sigctx = (P11PROV_SIG_CTX *)ctx;
 
-    P11PROV_debug(
-        "rsa digest sign final (ctx=%p, sig=%p, siglen=%zu, "
-        "sigsize=%zu)",
-        ctx, sig, *siglen, sigsize);
+    P11PROV_debug("rsa digest sign final (ctx=%p, sig=%p, siglen=%p, "
```

Comment:
Maybe we should just initialize the output to 0 rather then changing what we print ?

Reply:
Given that this is output variable, I am not sure what would be the point in initializing it to 0 and then printing it. Yeah, I agree that printing the pointer is not much more helpful, but I am ok with either direction (as long as we will not dereference null pointer or print uninitialized variables).

Reply:
Printing 0 after assigning it still tells the reader what values we have in there, better than printing a pointer. Doesn't really matter that we assigned that value.

Reply:
ok. Will do.